client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-manage-submitters/motion-manage-submitters.component.ts

```diff
@@ -91,20 +80,26 @@ export class MotionManageSubmittersComponent extends BaseUiComponent implements
     ) {
         super();
 
-        this.addSubmitterForm = new FormControl([]);
         this.editSubmitterObservable = this.editSubmitterSubject.asObservable();
     }
 
     public ngOnInit(): void {
         this.subscriptions.push(
-            this.addSubmitterForm.valueChanges.subscribe(nextValue => {
-                if (nextValue) {
-                    this.addUserAsSubmitter(this.userRepository.getViewModel(nextValue)!);
-                }
+            this.userRepository.getViewModelListObservable().subscribe(users => {
+                this.usersSubject.next(users);
+                this.filterUsers();
             })
         );
     }
 
+    /* Triggers an update of the filter for the list of available submitters
+     */
+    private filterUsers(): void {
+        const submitters = this.editSubmitterSubject.value.map(submitter => submitter.user_id ?? submitter.id);
+        const availableUsers = this.usersSubject.getValue().filter(user => !submitters.includes(user.id));
+        this.filteredUsersSubject.next(availableUsers);
+    }
+
```

Comment:
Can you provide those filter logic in the `UserCreateSearchSelectorComponent`?
So, the `UserCreateSearchSelectorComponent` gets a list of user-ids which are already taken and filters the list of available users/participants of its own.